### ISSUE TITLE
Tarantool couldn't start when space have FK constraints (vinyl)

### ISSUE THREAD

**Issue by kshcherbatov** [tarantool/tarantool#4190]
```
engine = 'vinyl'
box.execute('pragma sql_default_engine=\''..engine..'\'')
box.execute("CREATE TABLE parent (id INT PRIMARY KEY);")
box.execute("CREATE TABLE child (id INT PRIMARY KEY);")
box.execute("ALTER TABLE child ADD CONSTRAINT fk1 FOREIGN KEY (id) REFERENCES parent;")
box.execute("INSERT INTO parent VALUES (1), (2), (3);")
box.execute("INSERT INTO child VALUES (2);")
box.snapshot()
-- now restart Tarantool
2019-04-29 14:34:39.429 [24414] main/102/interactive alter.cc:4013 E> ER_CREATE_FK_CONSTRAINT: Failed to create foreign key constraint 'FK1': referencing space must be empty
2019-04-29 14:34:39.429 [24414] main/102/interactive F> can't initialize storage: Failed to create foreign key constraint 'FK1': referencing space must be empty
```

**Comment by kshcherbatov** [comment]
May be closed with #3699

**Comment by ImeevMA** [comment]
For 2.11 and later version the problem was fixed by closing #6986.

There are no fixes for 2.10 and earlier, and most likely there won't be. However, the FK constraints can be properly dropped if you choose to sacrifice them for the sake of running Tarantool. To do this before running box.cfg{} you need to execute this:
```
box.ctl.on_schema_init(function()
    box.space._fk_constraint:before_replace(function(o,n)
        if n ~= nil and box.space[n[3]].engine == 'vinyl' then
            return nil
        end
    end)
end)
```
